src/components/Spinner/index.tsx

```diff
@@ -6,10 +6,11 @@ import { color } from '../../theme'
 
 export interface SpinnerProps {
   size?: number
+  color?: string
 }
 
 const StyledSpinnerIcon = styled(SpinnerIcon)`
-  color: ${color.foreground};
+  color: ${props => props.color || color.invertedBackground};
```

Comment:
A color shouldn't be a background right? Why not default to `color.foreground`?

Reply:
@glenngijsberts the spinner then looks weird in an input component.. I'm wasn't quite sure to what it should default but foreground did not seem to be it.

Reply:
Yes, you're right!